state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1310,6 +1326,50 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(14)]
+		#[pallet::weight(T::WeightInfo::open_private_channel())]
+		pub fn open_private_channel(origin: OriginFor<T>) -> DispatchResult {
+			let broker_id = T::AccountRoleRegistry::ensure_broker(origin)?;
+
+			ensure!(
+				!BrokerPrivateChannels::<T, I>::contains_key(&broker_id),
+				Error::<T, I>::PrivateChannelExistsForBroker
+			);
+
+			// TODO: burn fee for opening a channel?
```

Comment:
I see this burning/rent is mentioned in the ticket, but also that this PR closes that ticket? so maybe we create another to address this separately.

Reply:
I will create tickets once we decide what should be in the scope of this PR and what isn't (I mentioned this in the description).

Reply:
> Closing private channels upon account deletion (maybe this isn't necessary for an MVP?)
(will create separate linear tickets for these if we agree to leave them out of this PR)

Ah ok, that seemed to suggest was just about the account deletion